eradiate/scenes/biosphere.py

```diff
@@ -134,17 +107,13 @@ class HomogeneousDiscreteCanopy(SceneElement):
             different ``seed`` values and after a certain amount of
             failures, run it without overlap avoidance.
 
-        Default value: True
+    ``leaf_reflectance`` (float or :class:`~eradiate.scenes.spectra.Spectrum`):
+        Reflectance spectrum of the leaves in the cloud. Must be a reflectance
+        spectrum (dimensionless). Default: 0.5.
```

Comment:
Why does it say "must be a reflectance spectrum" here? What does that sentence adds to the previous sentence, given that the accepted types are `float or :class:~eradiate.scenes.spectra.Spectrum`

Reply:
This means that the spectrum must have `self.quantity == PhysicalQuantity.REFLECTANCE`. I still have to find a way to properly document this, but I don't think I can do it as part of this PR.